docs/pages/guides/react-native-integration.mdx

```diff
@@ -187,6 +192,9 @@ yarn add @mobile-wallet-protocol/wagmi-connectors@latest
 
 Simply import the `createConnectorFromWallet` function and pass in the wallet you want to use to wagmi config.
 
+<Callout type="success" >
+A new config `appCustomScheme` is introduced to remove the Done screen after signing.
+</Callout>
```

Comment:
This looks the same as above. Redundant?

Reply:
we provide 2 interfaces here, some user may jump right into the second one, would like to emphasize in both section